<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamFormatPicker</class>
 <widget class="QWidget" name="StreamFormatPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>143</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QGroupBox exposes the title as QString (even if internally its probably a QLabel) */
QGroupBox {
	font-weight: bold;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="formatGroup" native="true">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	padding: 5px 15px 5px 15px;
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="defaultButton">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Simple</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/menu/youtube_48x48.png</normaloff>:/icons/menu/youtube_48x48.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Advanced:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="customAudioButton">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Audio</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/menu/stream_audio_16x16.png</normaloff>:/icons/menu/stream_audio_16x16.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="customVideoButton">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Video</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/menu/stream_video_16x16.png</normaloff>:/icons/menu/stream_video_16x16.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="choiceStackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="pageDefault">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="defaultSettingsGroup" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Detected media:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>511</width>
                 <height>323</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="detectedMediaCheckBoxList" native="true">
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QRadioButton" name="radioButton">
                     <property name="text">
                      <string notr="true">RadioButton</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_7">
                     <property name="text">
                      <string notr="true">RadioButton</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageCustom">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="audioGroup" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Audio:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="audioComboBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="videoGroup" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Video:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="videoComboBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="warningGroup" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>audio/video information is not available</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
